tests/all-fail/AllFailTest.bat

```diff
@@ -7,21 +7,26 @@ REM ---------------------------------------------------
     REM Initalize result variable
     set "slug=AllFail"
 
-    CALL :CheckEmptyFile
+    CALL :Initialize
 
     REM --------------------
     REM Test Case Start \/\/
     REM --------------------
     set "expected=true"
     set "if_success=Test passed"
-    set "if_failed=Test failed: The year should be divided into four."
+    set "if_failed=Test failed: '1996' should be divided into four."
     CALL :Assert 1996
 
     set "expected=false"
     set "if_success=Test passed"
-    set "if_failed=Test failed: 2015 shouldn't be divided into four."
+    set "if_failed=Test failed: '2015' shouldn't be divided into four."
     CALL :Assert 2015
 
+    set "expected=false"
+    set "if_success=Test passed"
+    set "if_failed=Test failed: Expected value to be '^%expected%', but got '^%stdout%'."
+    CALL :Assert 2017
```

Comment:
https://github.com/exercism/batch-test-runner/pull/2#issuecomment-1918635455

> It would be even better if there was something like: "Expected value to be X, but got Y." or something like that. Would that be possible?